src/rspamadm/commands.c

```diff
@@ -40,6 +41,7 @@ const struct rspamadm_command *commands[] = {
 	&fuzzyconvert_command,
 	&signtool_command,
 	&lua_command,
+    	&decrypt_header_command,
```

Comment:
Please refrain from adding rspamadm utilities in C. The proper way is to do it in Lua - you can check `lualib/rspamadm` for examples.